flake.nix

```diff
@@ -1,12 +1,11 @@
 {
- inputs = {
+  inputs = {
     holonix = {
-      url = "github:holochain/holonix?ref=main";
+      url = "github:holochain/holonix?ref=2d9ac2409a81ec67ed237e6f7a1ddcbe5f6d133b";
```

Comment:
was this intentional?

Reply:
Yes, this is temporary to let the workflow pass. It's a commit of Holonix that doesn't have the removals of DNA properties `origin_time` and `quantum_time` etc.